packages/bem-twig-extension/lib/bem-twig-extension.js

```diff
@@ -31,7 +31,7 @@ function bemTwigExtension(Twig) {
     }
     
     // If extra non-BEM classes are added.
-    if (extra.length && Array.isArray(extra)) {
+    if (extra.length && Array.isArray(extra) && extra != '') {
```

Comment:
You might be able to get what you want by just checking to see if extra is an array:

```javascript
if (Array.isArray(extra)) {
  classes = [
    ...classes,
    ...extra,
  ];
}
```


Also, hi friends, apparently I'm subscribed to this repo 😂